src/pretix/static/pretixcontrol/scss/main.scss

```
@@ -160,6 +160,9 @@ p.bigger {
 .helper-space-below {
     margin-bottom: 10px;
 }
+.helper-font-size-24 {
+    font-size: 24px;
+}
 
```

I don’t think using the absolut size in the utility classes name is a good idea. Boostrap3 already offers font-sizing classes .h1 to .h6 – problem is, that they also manipulate the margin. What about adding utility classes that are use the same naming scheme as already available? Like so:

```suggestion
.text-h1 { font-size: $font-size-h1; }
.text-h2 { font-size: $font-size-h2; }
.text-h3 { font-size: $font-size-h3; }
.text-h4 { font-size: $font-size-h4; }
.text-h5 { font-size: $font-size-h5; }
.text-h6 { font-size: $font-size-h6; }

```

In your case, you would need `.text-h3` which produces 24px font-size.

---

This is absolutely better, good suggestion :+1: